component/client/backgroundContext.tsx

```diff
@@ -1,7 +1,92 @@
 const backgroundContext: any = chrome.extension.getBackgroundPage();
 
-export const storage = backgroundContext.backgroundStorageService;
-export const room = backgroundContext.room;
+export interface IRoomService {
```

Comment:
Why is this within the `backgroundContext` file rather than a separate file? I think it is a bad idea to pollute a generalized file such as `backgroundContext`.